domenetjenester/brevbestiller/src/main/java/no/nav/foreldrepenger/melding/brevmapper/brev/opphørSVP/OpphørPeriodeMerger.java

```diff
@@ -0,0 +1,77 @@
+package no.nav.foreldrepenger.melding.brevmapper.brev.opphørSVP;
+
+import no.nav.foreldrepenger.melding.brevmapper.brev.felles.DatoVerktøy;
+import no.nav.foreldrepenger.melding.geografisk.Språkkode;
+import no.nav.foreldrepenger.melding.integrasjon.dokgen.dto.opphørSVP.OpphørPeriode;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class OpphørPeriodeMerger {
```

Comment:
Er alle scenarioer her dekket igjennom mapper-testen, eller er det noe som bør verifiseres med enhetstester direkte på denne klassen?

Reply:
jeg har laget enhetstester for denne klassen også, og fant et par feil